### ISSUE TITLE
Cannot build fedora rpm - setup.py cleans up build

### ISSUE THREAD

**Issue by opoplawski** [hpcugent/vsc-base#172]
The canonical way python rpms are made in Fedora is:
~~~~~
%build
%{__python2} setup.py build

%install
%{__python2} setup.py install --skip-build --root %{buildroot}
~~~~~
This breaks with vsc-base because the first thing the install line does is cleanup the build:
~~~~~
+ /usr/bin/python2 setup.py install --skip-build --root /export/home/orion/redhat/BUILDROOT/python-vsc-base-2.2.0-1.fc21.x86_64
cleanup build
~~~~~
I can workaround by dropping --skip-build, but it would be nice if this was supported.

**Comment by stdweird** [comment]
@opoplawski #173 should fix this

**Comment by boegel** [comment]
@opoplawski: just out of curiosity: why the interest in packaging `vsc-base`? Is this related to packaging EasyBuild, by any chance?

**Comment by boegel** [comment]
@opoplawski should be fixed in vsc-base v2.2.1, via #173, please give it a shot

**Comment by opoplawski** [comment]
#173 does seem to fix it.  Yes, I'm looking at packaging EasyBuild.

**Comment by boegel** [comment]
@JensTimmerman was also looking into this a while ago, not sure how far he got in the process...

**Comment by JensTimmerman** [comment]
I didn't get to the canonical  Fedora way yet, 

Creating rpm's out of EasyBuild isn't the hard part, getting through all of Fedora's guidelines, approval's, registration on koji,  finding a sponsor for your package etc, is what took me most time.
I have created and working source rpm's, but then stumbled on some configuration issues for EasyBuild, which were fixed in 2.0 I believe.
You can find my work source rpm's here: http://koji.fedoraproject.org/koji/tasks?owner=jenstimmerman&state=all
